backend/app/exceptions.py

```diff
@@ -272,3 +272,11 @@ class InvalidMatchedRewardsStrategy(OctantException):
 
     def __init__(self):
         super().__init__(self.description, self.code)
+
+
+class EstimateRewardsNotSupported(OctantException):
+    code = 500
```

Comment:
@kgarbacinski: why `500`? 500 means unexpected. It's not unexpected. I suggest 404, maybe.

Reply:
it must've slipped through the be's code review, just removed it and updated your branch